include/drivers/pcie/msi.h

```diff
@@ -17,6 +17,23 @@
 extern "C" {
 #endif
 
+#ifdef CONFIG_PCIE_CONTROLLER
+struct generic_msi_vector {
```

Comment:
msi_vector_generic_t ? 

I already noticed that having generic_ as a prefix is not great. Usually we have the domain first (pcie_ etc...).

Reply:
OK